### ISSUE TITLE
Cannot load Bible reference at the moment

### ISSUE THREAD

**Issue by kimberlyas** [bibleup/bibleup#42]
**Describe the bug**
The following message is being displayed on popup of Scriptures: *"Cannot load Bible reference at the moment"* for all versions available of the Bible. This bug is occurring on all devices and in all browsers. 
Getting a 403 Forbidden error in console for request to https://api.scripture.api.bible/...

**Comment by Bukunmikuti** [comment]
Thanks for the heads up @kimberlyas. I released [v1.0.3](https://github.com/bibleup/bibleup/releases/tag/v1.0.3) to address this
I also spotted a bug affecting the wiki popup style and this is also fixed in v1.0.4 (the latest version)
Please let me know if the latest version solved the issue for you

**Comment by kimberlyas** [comment]
Yes, thank you for resolving the issue @Bukunmikuti. The new update is working.

**Comment by Smoses00814** [comment]
I'm having the same issue with message _"Cannot load Bible reference at the moment."_  Wordpress only has v1.01 available for download. How do I upload/download v1.04?

**Comment by Bukunmikuti** [comment]
Hello @Smoses00814
The WordPress plugin will be updated to the latest very soon, before the weekend. Your issue might be a problem with cache. Please try with a different browser and see if it solves it. Please let me know if it works or not.

**Comment by Smoses00814** [comment]
Thank you. I was able to clear the cache and references are loading now. I look forward to the new version being available on WordPress.
